frontend/src/components/AddElection.tsx

```diff
@@ -82,7 +82,7 @@ const AddElection = () => {
             description: description, // mark-up text describing the election
             start_time: new Date(startDate),   // when the election starts 
             end_time: new Date(stopDate),   // when the election ends
-            owner_id: '0',
+            owner_id: authSession.getIdField('sub'),
```

Comment:
We'll want to verify that this matches the user id in the backend

Reply:
Yep, it matches up

<img width="735" alt="image" src="https://user-images.githubusercontent.com/9289903/156906025-ae064a90-6100-47f2-9c91-8093917449d1.png">